modules/understanding-upgrade-channels.adoc

```diff
@@ -10,14 +10,16 @@
 = {product-title} upgrade channels and releases
 
 In {product-title} 4.1, Red Hat introduced the concept of channels for
-recommending the appropriate release versions for cluster upgrade. By controlling
-the pace of upgrades, these upgrade channels allow you to choose an upgrade
-strategy. Upgrade channels are tied to a minor version of
-{product-title}. For instance, {product-title} {product-version}
-upgrade channels will never include an upgrade to a 4.5 release. This strategy ensures that
-administrators explicitly decide to upgrade to the next minor version of
-{product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the `openshift-install`
-binary file for a specific version of {product-title} always installs that version.
+recommending the appropriate release versions for cluster upgrade. By
+controlling the pace of upgrades, these upgrade channels allow you to choose an
+upgrade strategy. Upgrade channels are tied to a minor version of
+{product-title}. For instance, {product-title} {product-version} upgrade
+channels will never include an upgrade to a 4.6 release. This strategy ensures
```

Comment:
See my comment on this in another pending PR: https://github.com/openshift/openshift-docs/pull/25307#issuecomment-688885771. I believe this should be `4.7` (4.5 doc was outdated, which I think should be `4.6`). Waiting on confirmation though :slightly_smiling_face: